src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -213,7 +147,14 @@ function MoneyRequestPreview({
         }
 
         let message = translate('iou.cash');
-        if (ReportUtils.isPaidGroupPolicyExpenseReport(iouReport) && ReportUtils.isReportApproved(iouReport) && !ReportUtils.isSettled(iouReport?.reportID)) {
+        if (hasViolations && transaction) {
+            const violations = TransactionUtils.getTransactionViolations(transaction, transactionViolations);
+            if (violations?.[0]) {
+                const violationMessage = ViolationsUtils.getViolationTranslation(violations[0], translate);
+                const isTooLong = violations.filter((v) => v.type === 'violation').length > 1 || violationMessage.length > 15;
+                message += ` • ${isTooLong ? translate('violations.reviewRequired') : violationMessage}`;
+            }
+        } else if (ReportUtils.isPaidGroupPolicyExpenseReport(iouReport) && ReportUtils.isReportApproved(iouReport) && !ReportUtils.isSettled(iouReport)) {
```

Comment:
Similar comment, let's pass `iouReport?.reportID` and revert the change to `ReportUtils.isSettled`

Reply:
we need to pass `iouReport?.reportID ?? ""`, or else we need to modify the function to accept null values.

Reply:
And it's worth noting that all the other functions in the various Utils files all take the object and not just the ID.

```tsx
    const hasReceipt = TransactionUtils.hasReceipt(transaction);
    const isScanning = hasReceipt && TransactionUtils.isReceiptBeingScanned(transaction);
    const hasViolations = TransactionUtils.hasViolation(transaction, transactionViolations);
    const hasFieldErrors = TransactionUtils.hasMissingSmartscanFields(transaction);
    const shouldShowRBR = hasViolations || hasFieldErrors;
    const isDistanceRequest = TransactionUtils.isDistanceRequest(transaction);
    const isFetchingWaypointsFromServer = TransactionUtils.isFetchingWaypointsFromServer(transaction);
    const isCardTransaction = TransactionUtils.isCardTransaction(transaction);
    ```